docker/py3-native/verify.py

```diff
@@ -14,3 +14,6 @@
 
 import pdf2image
 print(f'Using poppler version: {pdf2image.pdf2image._get_poppler_version("pdftocairo")}')
+
+import dnspython
```

Comment:
this should automatically run ippysocks-py3's verify file since it has a symlink, so this shouldnt be needed

Reply:
[Done](https://github.com/demisto/dockerfiles/pull/32946/commits/eeda0eb6a4597bcc323650b6fbafa18884918424).